lang/en.json

```diff
@@ -891,5 +892,14 @@
     "appealingTheDecisionDetails": "<p class=\"govuk-body\">If your application is unsuccessful, you can appeal within 4 weeks of the decision by writing to the Head of Corporate Services.</p><p class=\"govuk-body\">If the original decision is upheld, it will be referred to the Public Guardian for confirmation.</p>",
     "removeTrustCorporation": "Remove trust corporation",
     "yesToRemoveTrustCorporation": "yes to remove the trust corporation",
-    "removeReplacementTrustCorporation": "Remove replacement trust corporation"
+    "removeReplacementTrustCorporation": "Remove replacement trust corporation",
+    "whatIsYourHomePostcode": "What is your home postcode?",
+    "homePostcode": "Home postcode",
+    "thisIsRequiredToCheckAndProtect": "This information is required to check your identity and protect against fraud.",
+    "yourHomeAddressWillNotBeShown": "Your home address will not be shown to the donor or appear on the completed LPA.",
+    "selectYourHomeAddress": "Select your home address",
+    "yourHomeAddress": "Your home address",
+    "changeHomeAddressLink": "Change<span class=\"govuk-visually-hidden\"> home address</span>",
+    "homeAddress": "Home address",
+    "goToDashboard": "Go to dashboard"
```

Comment:
https://opgtransform.atlassian.net/browse/MLPAB-1578 says "Return to dashboard" for this, but the other ticket says "Go to dashboard" 🤔

Reply:
Consistency FTW 🙌  will check in with design+content and see which is the winner.

Reply:
Confirmed we're going with `Go to dashboard` in all cases.